### PR TITLE
 HPCC4J-583 Update filespray and dfuxref clients to 9.6.x

### PR DESCRIPTION
**Pull request by rpastrana**
- Updates filespray wsdl2java maven profile

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [x] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).

## Checklist:
- [x] I have created a corresponding JIRA ticket for this submission
- [x] My code follows the code style of this project.
  - [ ] I have applied the Eclipse code-format template provided.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the HPCC Systems CONTRIBUTORS document (https://github.com/hpcc-systems/HPCC-Platform/wiki/Guide-for-contributors).
- [ ] The change has been fully tested:
  - [ ] This change does not cause any existing JUnits to fail.
  - [ ] I have include JUnit coverage to test this change
  - [ ] I have performed system test and covered possible regressions and side effects.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] This change fixes the problem, not just the symptom

### CODE REVIEW

**Comment by rpastrana** [comment]
@jpmcmu the second commit contains the rest of the changes needed to get in synch w/ ESP 9.6.x
Notice the manual changes to wssmc wrapper changes which illustrate the problem we discussed this morning.

**Comment by rpastrana** [comment]
@jpmcmu third commit include changes for wswu, I ended up having to manually change the WSDL

**Review by jpmcmu — COMMENTED** [review]
@rpastrana A few questions, only real issue is the manual changes need to the wrapper we discussed offline

File: wsclient/src/main/java/org/hpccsystems/ws/client/gen/axis2/wscloud/latest/WsCloudStub.java [hunk 1]
```diff
@@ -1,796 +1,796 @@
...
+        .getOptions()
+        .setTo(new org.apache.axis2.addressing.EndpointReference(targetEndpoint));
+    _serviceClient.getOptions().setUseSeparateListener(useSeparateListener);
+  }
+
+  /** Default Constructor */
+  public WsCloudStub(org.apache.axis2.context.ConfigurationContext configurationContext)
+      throws org.apache.axis2.AxisFault {
+
+    this(configurationContext, "http://eclwatch-66ff99685c-4bl4k:8888/WsCloud?ver_=1.01");
```
[the remark this answers is not in the document]
Comment (rpastrana) [reply wsclient/src/main/java/org/hpccsystems/ws/client/gen/axis2/wscloud/latest/WsCloudStub.java:146]: I think the path and parameters are used to route and annotate the request url
We have made attempts to cleanse the url in the WSDL in the past, and future improvements should attempt to automatically do so.

File: wsclient/src/main/java/org/hpccsystems/ws/client/wrappers/gen/wssmc/BrowseResourcesResponseWrapper.java [hunk 2]
```diff
@@ -48,88 +50,88 @@ public BrowseResourcesResponseWrapper( ArrayOfEspExceptionWrapper _exceptions, S
...
+    private void copy(org.hpccsystems.ws.client.gen.axis2.wssmc.latest.BrowseResourcesResponse raw) {
         if (raw == null)
             return;
 
         if (raw.getExceptions() != null)
-            this.local_exceptions = new ArrayOfEspExceptionWrapper( raw.getExceptions());
+            this.local_exceptions = new ArrayOfEspExceptionWrapper(raw.getExceptions());
         this.local_portalURL = raw.getPortalURL();
-        this.local_netAddress = raw.getNetAddress();
+        // this.local_netAddress = raw.getNetAddress();
```
[the remark this answers is not in the document]
Comment (rpastrana) [reply wsclient/src/main/java/org/hpccsystems/ws/client/wrappers/gen/wssmc/BrowseResourcesResponseWrapper.java:60]: apparently the service will populate either field based on the client version. We should not allow these types of changes going forward. For now, let's add logic to the wrapper to map the response value

**Comment by rpastrana** [comment]
@jpmcmu 4th commit manually reinstates the netaddress field which was dropped in the latest update

**Review by jpmcmu — APPROVED** [review]
@rpastrana looks good

**Pull request merged**